output/csv/output.go

```diff
@@ -97,6 +100,7 @@ func newOutput(params output.Params) (*Output, error) {
 			csvWriter:    stdoutWriter,
 			row:          make([]string, 3+len(resTags)+1),
 			saveInterval: saveInterval,
+			timeFormat:   timeFormat,
```

Comment:
```suggestion
			timeFormat: config.TimeFormat,
```
It seems we can avoid allocating the `timeFormat` variable

Reply:
Yep, nice tip will remove.